src/platform/datapath_winuser.c

```diff
@@ -269,6 +269,11 @@ typedef struct CXPLAT_SOCKET {
     //
     uint8_t Internal : 1;
 
+    //
+    // Flag indicates the binding is being used for PCP.
+    //
+    BOOLEAN PcpBinding : 1;
```

Comment:
Since all the other flags use uint8_t here, we should probably use that here as well. I'm not sure I trust the compiler to combine these, even if BOOLEAN is mapped to uint8_t.